src/cpp/rtps/security/SecurityManager.cpp

```diff
@@ -343,9 +343,21 @@ bool SecurityManager::init(
                     // Should be activated here, to enable encription buffer on created entities
                     throw true;
                 }
+                else
+                {
+                    if (access_plugin_ != nullptr && local_permissions_handle_ == nullptr)
+                    {
+                        EPROSIMA_LOG_ERROR(SECURITY, "Participant is not allowed with its own permissions file.");
+                    }
+
+                    if (crypto_plugin_ != nullptr && local_participant_crypto_handle_ == nullptr)
+                    {
+                        EPROSIMA_LOG_ERROR(SECURITY, "Participant cryptography could not be configured.");
+                    }
+                }
             }
 
-            throw false;
+            throw exception;
```

Comment:
In general cases, I do agree.
But in this case I think it is important. Currently, from user perspective, the creation of the participant fails and has no clue what happened (what I can say from my experience that is very annoying).

This is a tricky point. @MiguelCompany any thoughts or ideas on how it is done in other parts of the code?

Reply:
Friendly ping @MiguelCompany @EduPonz

Reply:
Adding the `EPROSIMA_LOG_ERROR` calls seems enough for me. I would keep the `throw false;` here, since the catch of `SecurityException` is designed for the initialization of the logging plugin